openmc/deplete/results.py

```diff
@@ -157,6 +158,59 @@ def get_atoms(
 
         return times, concentrations
 
+    def get_mass(self,
+        mat: typing.Union[Material, str],
+        nuc: str,
+        mass_units: str = "g",
+        time_units: str = "s"
+    ) -> Tuple[np.ndarray, np.ndarray]:
+        """Get mass of nuclides over time from a single material
+
+        .. versionadded:: 0.13.4
+
+        Parameters
+        ----------
+        mat : openmc.Material, str
+            Material object or material id to evaluate
+        nuc : str
+            Nuclide name to evaluate
+        mass_units : {"g", "g/cm3", "kg"}, optional
```

Comment:
```suggestion
        mass_units : {"g", "g/cm3", "kg", "kg/m3"}, optional
```
Is it worth adding another option for units here